app/forms/idv/state_id_form.rb

```diff
@@ -3,9 +3,9 @@ class StateIdForm
     include ActiveModel::Model
     include FormStateIdValidator
 
-    ATTRIBUTES = %i[first_name last_name dob state_id_address1 state_id_address2
-                    state_id_city state_id_zipcode state_id_jurisdiction
-                    state_id_state state_id_number same_address_as_id].freeze
+    ATTRIBUTES = %i[first_name last_name dob identity_doc_address1 identity_doc_address2
+                    identity_doc_city identity_doc_zipcode state_id_jurisdiction
```

Comment:
I think so, though I'm a little fuzzy on what eligibility means in this case. There is [this comment](https://github.com/18F/identity-idp/pull/8121#discussion_r1157385972) from @eileen-nava in another PR that explains the difference between `state_id_state` and `state_id_jurisdiction`. It also links to a jira explanation that I can't see yet, but is hopefully helpful.

The ticket says:

> Note: we won’t rename `state_id_jurisdiction` because it’s an old field that is used 
in many places throughout the codebase. We’re just renaming the new fields that we’ve
added as part of DAV.

Reply:
Correct!

`state_id_jurisdiction` = the state that issued the ID
`state_id_state` = the state in the state id address

Also, yes, `state_id_jurisdiction` is used to determine whether or not the state id was issued by a state under AAMVA's jurisdiction.

Reply:
💡 what if we renamed `state_id_state` to `state_id_address_state`?

Reply:
i think `state_id_state` is fine but wondering if maybe including the comment for `state_id_jurisdiction` in more places where it is present will help act as a reminder that that field is for the issuing state?